apps/mobile/src/features/Assets/components/Tokens/Tokens.container.tsx

```diff
@@ -36,17 +36,17 @@ export function TokensContainer() {
       <AssetsCard
         name={item.tokenInfo.name}
         logoUri={item.tokenInfo.logoUri}
-        description={`${formatValue(item.balance, item.tokenInfo.decimals as number)} ${item.tokenInfo.symbol}`}
+        description={`${formatVisualAmount(item.balance, item.tokenInfo.decimals as number)} ${item.tokenInfo.symbol}`}
         rightNode={
           <Text fontSize="$4" fontWeight={400} color="$color">
-            ${item.fiatBalance}
+            {formatCurrency(item.fiatBalance, 'usd')}
```

Comment:
can't we say that the `formatCurrenty` function would take `usd` by default?

Reply:
We currently only have USD, but we have a Settings screen where the user will be able to select the currency. So it makes no sense to enforce the default in the formatCurrency function as we will have to always get the currency selected in the settings.